tests/001_tests.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from django.db.models.query import QuerySet
+from users.models import Profile
+
+
+@pytest.mark.django_db(transaction=True)
+class TestProfile:
+    def test_profie_created(cls):
+        out = Profile.objects.all()
+        assert isinstance(out, QuerySet)
+        assert all(isinstance(p, Profile) for p in out)
+        profiles_names = []
+        for p in out:
```

Comment:
profile instead of `p`
also I would add a blank line before this for loop

Reply:
+1